pkg/controller/controller.go

```diff
@@ -94,17 +101,27 @@ func NewResizeController(
 		pvcRateLimiter, fmt.Sprintf("%s-pvc", name))
 
 	ctrl := &resizeController{
-		name:                   name,
-		resizer:                resizer,
-		kubeClient:             kubeClient,
-		pvSynced:               pvInformer.Informer().HasSynced,
-		pvcSynced:              pvcInformer.Informer().HasSynced,
-		claimQueue:             claimQueue,
-		volumes:                pvInformer.Informer().GetStore(),
-		claims:                 pvcInformer.Informer().GetStore(),
-		eventRecorder:          eventRecorder,
-		usedPVCs:               newUsedPVCStore(),
-		handleVolumeInUseError: handleVolumeInUseError,
+		name:                    name,
+		resizer:                 resizer,
+		kubeClient:              kubeClient,
+		pvSynced:                pvInformer.Informer().HasSynced,
+		pvcSynced:               pvcInformer.Informer().HasSynced,
+		vacSynced:               vacInformer.Informer().HasSynced,
```

Comment:
don't forget check it in WaitForCacheSync when feature-gate is on

Reply:
Oh good point, will add this.